store/src/java/com/zimbra/cs/store/helper/StoreManagerResetHelper.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2022 Synacor, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software Foundation,
+ * version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ * ***** END LICENSE BLOCK *****
+ */
+package com.zimbra.cs.store.helper;
+
+import com.zimbra.common.localconfig.ConfigException;
+import com.zimbra.common.localconfig.LC;
+import com.zimbra.common.localconfig.LocalConfig;
+import com.zimbra.common.service.ServiceException;
+import com.zimbra.common.util.StringUtil;
+import com.zimbra.common.util.ZimbraLog;
+import com.zimbra.cs.account.Provisioning;
+import com.zimbra.cs.store.StoreManager;
+import com.zimbra.soap.admin.type.StoreManagerRuntimeSwitchResult;
+import org.dom4j.DocumentException;
+
+import java.io.IOException;
+
+import static com.zimbra.common.localconfig.LC.zimbra_class_store;
+
+public class StoreManagerResetHelper {
+
+    public static final String STATUS_FAIL = "FAIL";
```

Comment:
Move these to an enum in the switch result status.

Reply:
Done